pkg/platform/packet/packet.go

```diff
@@ -88,6 +91,7 @@ type config struct {
 	ReservationIDs           map[string]string `hcl:"reservation_ids,optional"`
 	ReservationIDsDefault    string            `hcl:"reservation_ids_default,optional"`
 	CertsValidityPeriodHours int               `hcl:"certs_validity_period_hours,optional"`
+	ControllerCLCSnippets    []string          `hcl:"controller_clc_snippets,optional"`
```

Comment:
All variables here apply either to the controller or cluster, why use the `Controller` prefix?

I'm in doubt, it seems okay to be explicit too. Just wanted to know your reasoning in case there are very good reasons to make it so :)

Reply:
Reasoning here is that `Controller` word prefixed to `Count` and `Type` and also in aws.go Hence I went for consistency.